client/src/pages/OnBoarding/OnBoarding.js

```diff
@@ -0,0 +1,94 @@
+import React, { useRef } from "react";
+import { Paper, Grid, Button, makeStyles } from "@material-ui/core";
+import {
+  Route,
+  useHistory,
+  Switch,
+  Redirect,
+  useRouteMatch,
+} from "react-router-dom";
+
+import SetTimezoneUrl from "./SetTimezoneUrl";
+import ConnectGoogleCalendar from "./ConnectGoogleCalendar";
+import SetAvailability from "./SetAvailability";
+
+const OnBoarding = (props) => {
+  //styles
+  const classes = useStyles();
+
+  //data related variables
+  const url = useRef();
+  const timezone = useRef();
+  const days = useRef();
+  const startHour = useRef();
+  const finishHour = useRef();
+  const history = useHistory();
+  const match = useRouteMatch({ path: "/onboarding/:page" });
+  const page = match ? match.params.page : 0;
+
+  const FlipToNextPage = () => {
+    switch (page) {
+      case "1":
+        history.push("/onboarding/2");
+        break;
+      case "2":
+        history.push("/onboarding/3");
+        break;
+      default:
+        break;
+    }
+  };
+
```

Comment:
Try making a handler function here, or pass state setters to your components

Reply:
I realize it would be much easier to just have those three components manage their own data handling and thus no need for handler function here :P